### DISCUSSION TITLE
Support Youtube Vanced (Music)

### DISCUSSION THREAD

**Discussion by Vasmares** [BimmerGestalt/AAIdrive#670] [Ideas]
Hi,

as the normal Youtube App is only partially supported, as in you cant use the cars UI to search for tracks etc, maybe Youtube Vanced can?
https://youtubevanced.com/

Furthermore this version has no ads in it.

I am only suggesting it for music. 
There is a Youtube Vanced Music App as well, I dont know if that might fit better,

**Comment by Vasmares** [comment]
Its partially supported. But fully controlabe would be a treat.

![Screenshot_20230211_150206_AAIdrive](https://user-images.githubusercontent.com/10863304/218262180-1a39eaaf-726d-4d4c-a7e9-d48b991127ba.jpg)

**Comment by hufman** [comment]
It would be up to YouTube Vanced (Music) to provide this functionality, please ask that app's authors to remove the MediaBrowserService onGetRoot whitelist. Youtube itself doesn't provide browse functionality, so it would be harder for them to add such functionality in their mod, but Youtube Music might be easier for them to mod.
